### ISSUE TITLE
Feature Request: Font Colours

### ISSUE THREAD

**Issue by scheMeZa** [euvl/vue-js-toggle-button#102]
It would be nice if we could also change the font colour of the text for states, too.

**Comment by kcdrez-dwd** [comment]
Agreed. I can't really use this until the font colors can be changed.
Why have a feature to change the bg color but not font? Seems kinda silly to me. Even in the live demo the off state has a light gray with white font.

**Comment by JenniferCruz** [comment]
Sure we can! But it'd be _nice_.

**Comment by dd701116** [comment]
This issue should be closed, no ?
